File: Mathematical/Java/Sum_of_odd_and_even_elements.java
class Solution
{
    public int[] find_sum(int n)
    {
        // Code here
        int arr[] = {0,0};
        for (int i=1; i<=n; i++) {
            if (i%2==0) {
                arr[1] += i;
            }
            else {
                arr[0] += i;
            }
        }
        return arr;
    }
}

File: Solutions/Python/Sum_of_odd_and_even_elements.py
class Solution:
	def find_sum(self, n):
		# Code here
		sum = (n*(n+1))/2
        n = n//2
        even = ((2*n) + (n)*(n-1)) #sum of A.P. 2,4,6,....
        return [int(sum-even), int(even)]
